src/main/resources/config/liquibase/changelog/20190624165700_add_update_password_date_field.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="20190624165700" author="dtsarenko">
+        <addColumn tableName="jhi_user">
+            <column name="update_password_date" type="timestamp" defaultValueDate="${now}"/>
```

Comment:
setting to default value may lead to filling this column for all tenants with currdate. Please check if it won't affect their business logic